doc/release-notes/6.5-release-notes.md

```diff
@@ -0,0 +1,385 @@
+
+### Update Collection API Inheritance
+
+The update collection (dataverse) API endpoint has been updated to support an "inherit from parent" configuration for metadata blocks, facets, and input levels.
+
+Previously, not setting these fields meant using a copy of the settings from the parent collection, which could get out of sync. See also [the docs](https://guides.dataverse.org/en/6.5/api/native-api.html#update-a-dataverse-collection), #11018, and #11026.
+
+### isMetadataBlockRoot and isFacetRoot
+
+The JSON payload of the "get collection" endpoint has been extended to include properties isMetadataBlockRoot and isFacetRoot. See also [the docs](https://guides.dataverse.org/en/6.5/api/native-api.html#view-a-dataverse-collection), #11012, and #11013.
+
+### Whitespace Trimming When Loading Metadata Block TSV Files
+
+When loading custom metadata blocks using the `api/admin/datasetfield/load` API endpoint, whitespace can be introduced into field names. Whitespace is now trimmed from the beginning and end of all values read into the API before persisting them. See #10688 and #10696.
+
+### Image URLs from the Search API
+
+As of 6.4 (#10855) `image_url` is being returned from the Search API. The logic has been updated to only show the image if each of the following are true:
+
+1. The data file is not harvested
+2. A thumbnail is available for the data file
+3. If the data file is restricted, then the caller must have DownloadFile permission for the data file
+4. The data file is NOT actively embargoed
+5. The data file's retention period has NOT expired
+
+See also #10875 and #10886.
+
+### Metrics API Bug Fixes
+
+Two bugs in the Metrics API have been fixed:
+
+- The /datasets and /datasets/byMonth endpoints could report incorrect values if or when they have been called using the "dataLocation" parameter (which allows getting metrics for local, remote (harvested), or all datasets) as the metrics cache was not storing different values for these cases.
+
+- Metrics endpoints whose calculation relied on finding the latest published dataset version were incorrect if/when the minor version number was > 9.
+
+The upgrade instructions below include a step for clearing the metrics cache.
+
+See also #10379 and #10865.
+
+### API Tokens
+
+An optional query parameter called "returnExpiration" has been added to the `/api/users/token/recreate` endpoint, which, if set to true, returns the expiration time in the response. See [the docs](https://guides.dataverse.org/en/6.5/api/native-api.html#recreate-a-token), #10857 and #10858.
+
+The `/api/users/token` endpoint has been extended to support any auth mechanism for retrieving the token information. Previously this endpoint only accepted an API token to retrieve its information. Now it accepts any authentication mechanism and returns the associated API token information. See #10914 and #10924.
+
+## Settings Added
+
+- `:GlobusBatchLookupSize`
```

Comment:
Do we need a description/explanation here?

Reply:
We've always had a "settings added" section and I think it's helpful to the sysadmins to always have this list to go to, even if it's short this time.